<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>eoceanui</class>
 <widget class="QWidget" name="eoceanui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>277</width>
    <height>138</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EnOcean</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0">
    <widget class="QPushButton" name="TeachButtonOFF">
     <property name="text">
      <string>Teach OFF</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="TeachButtonClearLog">
     <property name="text">
      <string>Clear Log</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Available serial ports</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="TeachButtonON">
     <property name="text">
      <string>Teach ON</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="comboBoxComPort"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
